### PR TITLE
ramips: update Bolt BL201 to match stock bootloader

### PR DESCRIPTION
**Pull request by yHpgi**
### Comment:
No need to change the bootloader using breedweb anymore.
But breedweb can still be used.

### Changes:
1. Update device baudrate
2. Update some of leds and button gpios values and names
3. Update partition layout to match the stock layout
4. Update missing package depends for usb
5. Add device UIMAGE Magic value

### CODE REVIEW

File: target/linux/ramips/dts/mt7620a_bolt_bl201.dts [hunk 1]
```diff
@@ -3,94 +3,120 @@
...
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio2 4 GPIO_ACTIVE_LOW>;
 		};
 	};
 
 	keys {
 		compatible = "gpio-keys";
 
+		wps {
+			label = "WPS";
```
Comment (1715173329) [inline target/linux/ramips/dts/mt7620a_bolt_bl201.dts:111]:
```suggestion
			label = "wps";
```
please use "standard" label as much as possible, i.e. `wps`.

File: target/linux/ramips/dts/mt7620a_bolt_bl201.dts [hunk 2]
```diff
@@ -3,94 +3,120 @@
...
+			linux,code = <KEY_WPS_BUTTON>;
+			gpios = <&gpio1 1 GPIO_ACTIVE_LOW>;
+		};
+
 		reset {
-			label = "reset";
-			gpios = <&gpio1 26 GPIO_ACTIVE_LOW>;
+			label = "RST";
```
Comment (1715173329) [inline target/linux/ramips/dts/mt7620a_bolt_bl201.dts:117]:
```suggestion
			label = "reset";
```

File: target/linux/ramips/dts/mt7620a_bolt_bl201.dts [hunk 3]
```diff
@@ -196,7 +223,7 @@
 };
 
 &ethernet {
-	nvmem-cells = <&macaddr_factory_28 0>;
+	nvmem-cells = <&macaddr_factory_28>;
```
Comment (1715173329) [inline target/linux/ramips/dts/mt7620a_bolt_bl201.dts:226]:
```suggestion
	nvmem-cells = <&macaddr_factory_28 0>;
```

please keep it as-is. it's required by `mac-base` binding.

File: target/linux/ramips/dts/mt7620a_bolt_bl201.dts [hunk 4]
```diff
@@ -3,94 +3,120 @@
...
 
-		wan {
-			function = LED_FUNCTION_WAN;
-			color = <LED_COLOR_ID_BLUE>;
-			gpios = <&gpio1 5 GPIO_ACTIVE_LOW>;
-		};
-
-		power_red: power_red {
-			function = LED_FUNCTION_POWER;
+		led_reset: reset {
```
Comment (1715173329) [inline target/linux/ramips/dts/mt7620a_bolt_bl201.dts:27]:
```suggestion
		led_reset: led-reset {
```

File: target/linux/ramips/dts/mt7620a_bolt_bl201.dts [hunk 5]
```diff
@@ -3,94 +3,120 @@
...
+			function = LED_FUNCTION_BOOT;
 			color = <LED_COLOR_ID_RED>;
 			gpios = <&gpio1 8 GPIO_ACTIVE_LOW>;
 		};
 
-		power_blue: power_blue {
+		led_power: power {
```
Comment (1715173329) [inline target/linux/ramips/dts/mt7620a_bolt_bl201.dts:33]:
```suggestion
		led_power: led-power {
```

File: target/linux/ramips/dts/mt7620a_bolt_bl201.dts [hunk 6]
```diff
@@ -3,94 +3,120 @@
...
 			function = LED_FUNCTION_POWER;
 			color = <LED_COLOR_ID_BLUE>;
 			gpios = <&gpio1 9 GPIO_ACTIVE_LOW>;
 		};
 
-		lte_s1_blue {
-			label = "blue:lte_s1";
+		led_lte1 {
```
Comment (1715173329) [inline target/linux/ramips/dts/mt7620a_bolt_bl201.dts:39]:
```suggestion
		led-lte1 {
```

for node name, dash `-` is used instead of underline `_`.

**Review by 1715173329 — CHANGES_REQUESTED** [review]

**Comment by yHpgi** [comment]
@1715173329 Resolved.

**Review by 1715173329 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**